python/lsst/obs/decam/ingestCalibs.py

```diff
@@ -113,9 +114,16 @@ def translate_filter(self, md):
             FITS header metadata.
         """
         if md.exists("FILTER"):
-            if md.exists("OBSTYPE") and "zero" in md.getScalar("OBSTYPE").strip().lower():
-                return "NONE"
-            return CalibsParseTask.translate_filter(self, md)
+            if md.exists("OBSTYPE"):
+                obstype = md.getScalar("OBSTYPE").strip().lower()
+                if "zero" in obstype or "bias" in obstype:
+                    return "NONE"
+            filterName = CalibsParseTask.translate_filter(self, md)
+            # Workaround if/else that can be removed when DM-24514 is complete
+            if filterName == '_unknown_' and md.exists("CALIB_ID"):
```

Comment:
In most cases you can think of a PropertyList as a dict and use normal dict code. So `"CALIB_ID" in md` and  `md["OBSTYPE"]`